src/functionalTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/ParentIntegrationTestSupport.java

```diff
@@ -151,6 +152,11 @@ static void validateDbRecordCountFor(JdbcTemplate jdbcTemplate, String queryName
         assertEquals(expectedCount, jdbcTemplate.queryForList(queryName).size());
     }
 
+    static void validateDbRecordValuesFor(JdbcTemplate jdbcTemplate, String queryName, String columnName) {
+        Map<String, Object> hmValue = jdbcTemplate.queryForList(queryName).get(0);
+        assertNull(hmValue.get(columnName));
```

Comment:
Can you assert for the presence of data instead of null?

Reply:
done